clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/data/InstanceFamilyUtils.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.aws.data;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import java.util.Map;
+import java.util.Set;
+
+public class InstanceFamilyUtils {
```

Comment:
There is a bit of overlap between what this is providing and what is in `BlockDeviceConfig.groovy` - it would be great to only have one spot to manage hard coded lists of instance type specific config

Reply:
`BlockDeviceConfig.groovy` does have a mapping of instance type to block devices. However, looks like the class is well defined with a single purpose. 
The new `InstanceFamilyUtils.java`'s purpose is to the gather utility functions at the instance family granularity, that are currently in `BasicAmazonDeployHandler`. This helps to reuse the validations for burstable instance types in `BasicAmazonDeployDescriptionValidator`. 

I could refactor these 2 classes into a new `AwsConfigurationHelper` with `BlockDeviceConfig` inner class and all utility functions. Do you think this makes sense?

Reply:
yeah - I like the name InstanceFamilyUtils since everything it does is related to instance types and BlockDeviceConfig could fit under that

Reply:
👍